<launch>

  <node pkg="nodelet" type="nodelet" name="cloud_nodelet"  args="manager" output="screen"  />

    <!--TODO: options for kinect and asus -->
  <!--include file="$(find baxter_demos)/launch/freenect_nodelet.launch"/-->
  
  <!--include file="$(find baxter_demos)/launch/openni2_nodelet.launch"/-->

  <node pkg="nodelet" type="nodelet" name="CloudSegmenter" args="load baxter_demos/CloudSegmenter cloud_nodelet" output="screen">
  </node>

  <node pkg="baxter_demos" type="ColorPicker" name="color_picker" output="screen"/>

    <rosparam command="load" file="$(find baxter_demos)/config/object_finder_3d.yaml"/>
</launch>
